app/src/main/java/edu/mills/findeatfood/ResultsFragment.java

```diff
@@ -0,0 +1,131 @@
+package edu.mills.findeatfood;
+
+import android.app.Activity;
+import android.app.ListFragment;
+import android.app.ProgressDialog;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import edu.mills.findeatfood.models.Matches;
+import edu.mills.findeatfood.models.SearchRecipes;
+
+/**
+ * Shows list of recipe results returned from the API. Clicking the recipe will launch
+ * {@link RecipeDetailFragment}.
+ */
+public class ResultsFragment extends ListFragment {
+
+    private ResultsListListener listener;
+    private List<Matches> recipesList = new ArrayList<Matches>();
+    private List<String> ingredients = new ArrayList<String>();
+    private List<String> dietRestrictions = new ArrayList<String>();
+    private List<String> allergyRestrictions = new ArrayList<String>();
+    private ProgressDialog progress;
+    private int startPosition;
+
+    /**
+     * Constructor for results fragment. This constructor uses
```

Comment:
Implementation details (methods called) don't belong in javadoc. Instead, you can say that the fragment retains the list of recipes upon device rotation. That's really what the caller needs to know.